aws-integration/README.md

```diff
@@ -0,0 +1,138 @@
+
+- Kyma as the target deployment environment
+- AWS account with permissions to create new users and security policies
+- Helm 3.x if you want to deploy open-telemetry sample application
+
+## Installation
+
+### Preparation
+
+1. Export your Namespace as a variable. Replace `{NAMESPACE}` placeholder in the following command and run it:
+
+    ```bash
+    export KYMA_NS="{NAMESPACE}"
+    ```
+1. If you don't have created a Namespace yet, do it now:
+    ```bash
+    kubectl create namespace $KYMA_NS
+    ```
+1. Export the AWS region you want to use as variable. Replace `{NAMESPACE}` placeholder in the following command and run it:
+    ```bash
+    export AWS_REGION="{NAMESPACE}"
+    ```
+
+### Create AWS IAM User
+
+As a first step, we need to create an IAM User assign to it specific IAM policies through which our OTEL Collector will be communicating with AWS. 
+
+Firstly, we need to create IAM policy for pushing our metrics:
+1. Go to the AWS searchbar, and search for IAM service
+1. Go to the `Policies` section. and click `Create policy`
+1. Remove the `Popular services` flag, and select `CloudWatch` service. 
+1. Now, select `GetMetricData`, `PutMetricData`, `ListMetrics` actions and click `Next`
+1. Enter the policy name and click `Create policy`
+
+Now, we should create the policy for CloudWatch Logs:
+1. Repeat the first two steps you did while creating the policy for metrics
+1. Remove the `Popular services` flag, and select `CloudWatch Logs` service.
+1. Now, select `CreateLogGroup`, `CreateLogStream`, `PutLogEvents` actions and click `Next`
+1. Specify resource ARN for selected actions
+1. Enter the policy name and click `Create policy` 
+
+After creating the IAM Policies, we can finally create an IAM User:
+1. Go to the `Users` section and click `Add user`
+1. Enter the user name and click next
+1. Click to the `Attach policies directly`
+1. Select two new policies you added on previous steps as well as the `AWSXrayWriteOnlyAccess`
+1. Click `Next` and then `Create User`
+1. Open the new user
+1. Go to the `Security credentials` tab and click `Create access key`
+1. Select `Application running outside AWS` and then click `Next`
+1. Describe the purpose of this access key and click `Create access key`
+1. Now copy and save `Access key` and `Secret access key`
+1. Encode the keys into base64 encoding
+
+### Create a secret with AWS Credentials
+
+In order to connect OTEL Collector to AWS we need to define security credentials in the kyma system. 
+
+1. In the [values.yaml](./aws-secret/values.yaml), replace the `{KEY_ID_BASE64}` and `{KEY_SECRET_BASE64}` to your encoded access keys 
+2. Now, create the secret by using 
+    ```bash
+    kubectl apply -f ./aws-secret/values.yaml
+    ```
+
+### Deploy an OTEL Collector
+
+After creating a secret and configuering AWS, we can finally deploy an Otel Collector itself.
+
+1. Deploy an OTEL Collector by calling 
+    ```bash
+    kubectl apply -f ./aws-otel-collector/values.yaml
```

Comment:
Why it is called values.yaml? Confuses with a helm deployment. Isn't there a helm chart or a deployment.yaml available online to re-use instead of copying?